src/components/AboutModal/AboutModal.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Modal } from '../../index';
+
+const AboutModal = ({
+  children,
+  className,
+  show,
+  onHide,
+  productTitle,
+  logo,
+  altLogo,
+  trademarkText,
+  closeText,
+  ...rest
+}) => {
+  return (
+    <Modal
+      className={className}
+      contentClassName="about-modal-pf"
+      show={show}
+      onHide={onHide}
+      {...rest}
+    >
+      <Modal.Header>
+        <Modal.CloseButton onClick={onHide} closeText={closeText} />
+      </Modal.Header>
+      <Modal.Body>
+        <h1>{productTitle}</h1>
+        {children}
+        <div className="trademark-pf">{trademarkText}</div>
+      </Modal.Body>
```

Comment:
Not necessarily a fan of passing ```string``` props/rest/attributes in instead of letting it fall under ```{children}``` but I see why. This is one of those walk the line scenarios between stateless and opinionated, whereas we're producing an opinion about the About modal.

Reply:
agreed... i guess with extensions like this, my general hope is that these helper functions reduce effort in many use cases. I like the fact we are exposing all of the Modal components and sub components (and same w/ About Modal/About Modal sub components), and one could easily rebuild this wrapper if needed. It's nice having the flexibility downstream to choose common use cases or the option to build new patterns using the smaller Modal building blocks. The more common building blocks _and_ common patterns we can expose/reuse, the better adoption we should see i.m.h.o.